WordPress/Classes/ViewRelated/System/WPTabBarController.m

```diff
@@ -558,11 +569,25 @@ - (void) hideReaderBadge:(NSNotification *)notification
 
 - (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(NSDictionary *)change context:(void *)context
 {
-    if ([keyPath isEqualToString:WPApplicationIconBadgeNumberKeyPath]) {
+    if (object == self.tabBar && [keyPath isEqualToString:WPTabBarFrameKeyPath]) {
+        [self notifyOfTabBarHeightChangeIfNeeded];
```

Comment:
There's a typo that's causing a build error 👀 

```suggestion
        [self notifyOfTabBarHeightChangedIfNeeded];
```

Reply:
Fixed in 2fe04939665f11d78e5a62aa89d711fe505c2d11 🙈